src/dump.c

```diff
@@ -380,11 +380,11 @@ static void jl_serialize_module(jl_serializer_state *s, jl_module_t *m)
     write_uint8(s->s, m->infer);
 }
 
-static void jl_serialize_value_(jl_serializer_state *s, jl_value_t *v, int as_literal) JL_GC_DISABLED
+static inline int jl_serialize_generic(jl_serializer_state *s, jl_value_t *v) JL_GC_DISABLED
```

Comment:
Why is this declared inline?

Reply:
Also could use a comment saying what the return value means.

Reply:
It grew after I added the inline annotation. It's probably too big now to be inline